lib/collection.go

```diff
@@ -42,8 +42,29 @@ func (m CollectionMethods) Attributes() map[string]AttributeSet {
 // ErrListWarning is a warning that can occur while listing
 var ErrListWarning = base.ErrUnlistableReferences
 
+// CollectionListParams defines parameters for listing a user's collection
+type CollectionListParams struct {
+	params.List `json:"list,omitempty"`
```

Comment:
I could be wrong here, I forget how it works exactly, but will adding this json tag add another level to the serialized json? Would be cool if we could avoid that, if possible. We would like to get:

```
{
  "limit": ...
  "offset": ...
  "username": ...
  ...
}
```

rather than

```
{
  "list": {
    "limit": ...
    "offset": ...
  },
  "username": ...
  ...
}
```

Pretty sure that leaving off this json tag will get the former instead of the latter.

Reply:
you are 100% correct, and thank you for catching this!!!!!